src/components/Reports/FourteenMonthReports/Layout/Table/Table.tsx

```diff
@@ -124,69 +119,74 @@ export const FourteenMonthReportTable: React.FC<
           onRequestSort={onRequestSort}
         />
         <TableBody>
-          {orderedContacts?.map((contact) => (
-            <TableRow
-              key={contact.id}
-              hover
-              data-testid="FourteenMonthReportTableRow"
-            >
-              <StyledTableCell>
-                <Box display="flex" flexDirection="column">
-                  <Box display="flex" alignItems="center">
-                    {!isExpanded && <StyledInfoIcon fontSize="small" />}
-                    <NameTypography variant="body1" expanded={isExpanded}>
-                      <StyledLink
-                        onClick={() => onSelectContact(contact.id)}
-                        onMouseEnter={preloadContactsRightPanel}
-                        underline="hover"
-                      >
-                        {contact.name}
-                      </StyledLink>
-                    </NameTypography>
+          {orderedContacts?.map((contact) => {
+            const contactUrl = getContactUrl(contact.id);
+            return (
+              <TableRow
+                key={contact.id}
+                hover
+                data-testid="FourteenMonthReportTableRow"
+              >
+                <StyledTableCell>
+                  <Box display="flex" flexDirection="column">
+                    <Box display="flex" alignItems="center">
+                      {!isExpanded && <StyledInfoIcon fontSize="small" />}
+                      <NameTypography variant="body1" expanded={isExpanded}>
+                        <NextLink
+                          href={contactUrl}
```

Comment:
Another link that now is always underlined instead of on hover. Maybe we should add `defaultProps: { underline: 'hover' }` to `MuiLink` in the theme to help with consistency.